### PR TITLE
Add toggle state for intake piston

### PR DESCRIPTION
**Pull request by AndrewLester**
I feel like it shouldn't have taken an entire command class to do this but it did.

### CODE REVIEW

File: src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.IntakeSubsystem;
+
+public class ToggleIntakePistonCommand extends CommandBase {
+    private IntakeSubsystem intakeSubsystem;
+
+    public ToggleIntakePistonCommand(IntakeSubsystem intakeSubsystem) {
+        this.intakeSubsystem = intakeSubsystem;
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java:10]: requires?

File: src/main/java/frc/robot/RobotContainer.java [hunk 2]
```diff
@@ -203,8 +204,8 @@ private void configureButtonBindings() {
             new AutomaticShootCommand(TARGET_VELOCITY, NUM_BALLS_LOADED, shooterSubsystem)
         );
 
-        btnIntakeSolenoid.whenPressed(new InstantCommand(intakeSubsystem::extend, intakeSubsystem))
-        .whenInactive(new InstantCommand(intakeSubsystem::retract, intakeSubsystem));
+        btnIntakeSolenoid.toggleWhenPressed(new ToggleIntakePistonCommand(intakeSubsystem), true);
+
```
Comment (ErikBoesen) [inline src/main/java/frc/robot/RobotContainer.java]: Extra whitespace

**Review by ErikBoesen — CHANGES_REQUESTED** [review]

File: src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+    }
+
+    @Override
+    public void initialize() {
+        this.intakeSubsystem.extend();
```
Comment (ArchdukeTim) [inline src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java:15]: The command is called toggle but this only extends.

Comment (AndrewLester) [reply src/main/java/frc/robot/commands/ToggleIntakePistonCommand.java:15]: Good point, the command actually just retracts the intake when it ends, so it functions as an argument to `toggleWhenPressed`. It's a little weird but it works nicely when tested.

**Comment by ErikBoesen** [comment]
God, I always hate that extra whitespac

**Pull request merged**